lib/utils/rabbitmq/src/test/consumer-test.js

```diff
@@ -210,6 +210,29 @@ describe('Consumer', () => {
             assert.notCalled(amqpChannel.nack);
           });
         });
+
+        context('when message has no metadata', () => {
+          beforeEach(async() => {
+            fakeHandler.handle.returns(Promise.reject(new Error()));
+            fakeParser.toJSON.returns(JSON.parse('{ "someProp": "some val" }'));
```

Comment:
do you need to parse string to json in order to return it? can't you just return an object - { someProp: 'some val'}